bundle.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#run as normal user
+umask 002
+
+#make build dir
+echo "Creating build/js dir"
+mkdir -p build/js
+
+#bundle js
+echo "Creating bundle.."
+cat \
+  "blockly_compressed.js" \
+  "blocks_compressed.js" \
+  "blocks/coins.js" \
+  "javascript_compressed.js" \
+  "php_compressed.js" \
+  "msg/js/en.js" \
+  > build/js/coins-blockly.js
+echo "SUCCESS: build/js/coins-blockly.js"
```

Comment:
How do we move the built js file into coins_core? manually? 
The vendor dir is gitignored under coins_core, so that means , when going into production, we need to manually move the file as well?

Reply:
I committed it into coins_core https://github.com/MRN-Code/coins_core/pull/371/commits/5244e5ab2a8fa326e334cf572deaad991e9ec1b7

Reply:
I'm not really sure why the vender dir is gitignored. 🤔

In the coins_core PR I manually committed  `vendor/coins-blockly.js` so now it will be tracked even tho the `vendor` dir is gitignored.

Reply:
hmm, that `vendor` is different from `js/vendor` , the latter is not gitignored. So i think we are good.